Error when compiling CustomRow with Swift 3.0

- Environment: Xcode 8, and have tried the "2.0.0 Beta 1" and the latest commit from "master".

I have couple custom Rows that inherit from _FieldRow. and I'm getting a compiler error:

```
Undefined symbols for architecture i386:
  "direct field offset for Eureka._FieldCell.(textField.storage in _3A88909586E8B5678F695AC90DD64A31) : __ObjC.UITextField?", referenced from:
      Eureka._FieldCell.textField.setter : __ObjC.UITextField in IntAsStringRow.o
      Eureka._FieldCell.(textField.materializeForSet : __ObjC.UITextField).(closure #1) in IntAsStringRow.o
      Eureka._FieldCell.textField.setter : __ObjC.UITextField in IntCurrencyRow.o
      Eureka._FieldCell.(textField.materializeForSet : __ObjC.UITextField).(closure #1) in IntCurrencyRow.o
      Eureka._FieldCell.textField.setter : __ObjC.UITextField in IntAsStringRow.o
      Eureka._FieldCell.(textField.materializeForSet : __ObjC.UITextField).(closure #1) in IntAsStringRow.o
      Eureka._FieldCell.textField.setter : __ObjC.UITextField in IntCurrencyRow.o
      Eureka._FieldCell.(textField.materializeForSet : __ObjC.UITextField).(closure #1) in IntCurrencyRow.o
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Any Ideas on how I can fix this?

Thanks!

--- a/Source/Rows/DatePickerRow.swift
+++ b/Source/Rows/DatePickerRow.swift
@@ -26,18 +26,18 @@ import Foundation
 
 open class DatePickerCell : Cell<Date>, CellType {
     
-    open lazy var datePicker: UIDatePicker = { [unowned self] in
-        let picker = UIDatePicker()
-        picker.translatesAutoresizingMaskIntoConstraints = false
-        self.contentView.addSubview(picker)
-        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|-0-[picker]-0-|", options: [], metrics: nil, views: ["picker": picker]))
-        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-0-[picker]-0-|", options: [], metrics: nil, views: ["picker": picker]))
-        picker.addTarget(self, action: #selector(DatePickerCell.datePickerValueChanged(_:)), for: .valueChanged)
-        return picker
-        }()
+    public var datePicker: UIDatePicker
     
     public required init(style: UITableViewCellStyle, reuseIdentifier: String?){
+        self.datePicker = UIDatePicker()
+        self.datePicker.translatesAutoresizingMaskIntoConstraints = false
+        
         super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        self.contentView.addSubview(self.datePicker)
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|-0-[picker]-0-|", options: [], metrics: nil, views: ["picker": self.datePicker]))
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-0-[picker]-0-|", options: [], metrics: nil, views: ["picker": self.datePicker]))
+        self.datePicker.addTarget(self, action: #selector(DatePickerCell.datePickerValueChanged(_:)), for: .valueChanged)
     }
     
     required public init?(coder aDecoder: NSCoder) {

--- a/Source/Rows/PickerRow.swift
+++ b/Source/Rows/PickerRow.swift
@@ -28,19 +28,19 @@ import Foundation
 
 open class PickerCell<T> : Cell<T>, CellType, UIPickerViewDataSource, UIPickerViewDelegate where T: Equatable{
     
-    open lazy var picker: UIPickerView = { [unowned self] in
-        let picker = UIPickerView()
-        picker.translatesAutoresizingMaskIntoConstraints = false
-        self.contentView.addSubview(picker)
-        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|-0-[picker]-0-|", options: [], metrics: nil, views: ["picker": picker]))
-        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-0-[picker]-0-|", options: [], metrics: nil, views: ["picker": picker]))
-        return picker
-        }()
+    public var picker: UIPickerView
     
     private var pickerRow : _PickerRow<T>? { return row as? _PickerRow<T> }
     
     public required init(style: UITableViewCellStyle, reuseIdentifier: String?){
+        self.picker = UIPickerView()
+        self.picker.translatesAutoresizingMaskIntoConstraints = false
+        
         super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        self.contentView.addSubview(self.picker)
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "V:|-0-[picker]-0-|", options: [], metrics: nil, views: ["picker": self.picker]))
+        self.contentView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-0-[picker]-0-|", options: [], metrics: nil, views: ["picker": self.picker]))
     }
     
     required public init?(coder aDecoder: NSCoder) {

--- a/Source/Rows/StepperRow.swift
+++ b/Source/Rows/StepperRow.swift
@@ -15,6 +15,13 @@ open class StepperCell : Cell<Double>, CellType {
     public typealias Value = Double
     
     required public init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        self.stepper = UIStepper()
+        self.stepper.translatesAutoresizingMaskIntoConstraints = false
+
+        self.valueLabel = UILabel()
+        self.valueLabel.translatesAutoresizingMaskIntoConstraints = false
+        self.valueLabel.numberOfLines = 1
+
         super.init(style: style, reuseIdentifier: reuseIdentifier)
         height = { BaseRow.estimatedRowHeight }
     }
@@ -23,18 +30,9 @@ open class StepperCell : Cell<Double>, CellType {
         fatalError("init(coder:) has not been implemented")
     }
     
-    open lazy var stepper: UIStepper = {
-        let s = UIStepper()
-        s.translatesAutoresizingMaskIntoConstraints = false
-        return s
-    }()
+    public var stepper: UIStepper
     
-    open lazy var valueLabel: UILabel = {
-        let l = UILabel()
-        l.translatesAutoresizingMaskIntoConstraints = false
-        l.numberOfLines = 1
-        return l
-    }()
+    public var valueLabel: UILabel
     
     open override func setup() {
         super.setup()

--- a/Source/Rows/TextAreaRow.swift
+++ b/Source/Rows/TextAreaRow.swift
@@ -51,6 +51,9 @@ extension AreaCell {
 open class _TextAreaCell<T> : Cell<T>, UITextViewDelegate, AreaCell where T: Equatable, T: InputTypeInitiable {
     
     required public init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        self.textView = UITextView()
+        self.textView.translatesAutoresizingMaskIntoConstraints = false
+
         super.init(style: style, reuseIdentifier: reuseIdentifier)
     }
     
@@ -66,11 +69,7 @@ open class _TextAreaCell<T> : Cell<T>, UITextViewDelegate, AreaCell where T: Equ
         return v
     }()
     
-    open lazy var textView : UITextView = {
-        let v = UITextView()
-        v.translatesAutoresizingMaskIntoConstraints = false
-        return v
-    }()
+    public var textView: UITextView
     
     open var dynamicConstraints = [NSLayoutConstraint]()
     

--- a/Source/Rows/TextAreaRow.swift
+++ b/Source/Rows/TextAreaRow.swift
@@ -54,6 +54,11 @@ open class _TextAreaCell<T> : Cell<T>, UITextViewDelegate, AreaCell where T: Equ
         self.textView = UITextView()
         self.textView.translatesAutoresizingMaskIntoConstraints = false
 
+        self.placeholderLabel = UILabel()
+        self.placeholderLabel.translatesAutoresizingMaskIntoConstraints = false
+        self.placeholderLabel.numberOfLines = 0
+        self.placeholderLabel.textColor = UIColor(white: 0, alpha: 0.22)
+
         super.init(style: style, reuseIdentifier: reuseIdentifier)
     }
     
@@ -61,13 +66,7 @@ open class _TextAreaCell<T> : Cell<T>, UITextViewDelegate, AreaCell where T: Equ
         fatalError("init(coder:) has not been implemented")
     }
     
-    open lazy var placeholderLabel : UILabel = {
-        let v = UILabel()
-        v.translatesAutoresizingMaskIntoConstraints = false
-        v.numberOfLines = 0
-        v.textColor = UIColor(white: 0, alpha: 0.22)
-        return v
-    }()
+    public var placeholderLabel : UILabel
     
     public var textView: UITextView
     

